tensorflow/lite/kernels/internal/optimized/optimized_ops.h

```diff
@@ -3465,6 +3465,57 @@ void BroadcastDiv4DSlow(const ArithmeticParams& params,
   }
 }
 
+// TODO(benoitjacob): BroadcastDiv is intentionally duplicated from
+// reference_ops.h. For more details see the comment above the generic
+// version of BroadcastDiv4DSlow.
+inline void BroadcastDiv4DSlow(const ArithmeticParams& params,
+                               const RuntimeShape& unextended_input1_shape,
+                               const uint8* input1_data,
+                               const RuntimeShape& unextended_input2_shape,
+                               const uint8* input2_data,
+                               const RuntimeShape& unextended_output_shape,
+                               uint8* output_data) {
+  TFLITE_DCHECK_LE(unextended_input1_shape.DimensionsCount(), 4);
+  TFLITE_DCHECK_LE(unextended_input2_shape.DimensionsCount(), 4);
+  TFLITE_DCHECK_LE(unextended_output_shape.DimensionsCount(), 4);
+  const RuntimeShape output_shape =
+      RuntimeShape::ExtendedShape(4, unextended_output_shape);
+
+  NdArrayDesc<4> desc1;
+  NdArrayDesc<4> desc2;
+  NdArrayDescsForElementwiseBroadcast(unextended_input1_shape,
+                                      unextended_input2_shape, &desc1, &desc2);
+  
+  const float scale = static_cast<float>(
```

Comment:
Usage of floating point arithmetic in quantized ops is generally discouraged, but not a show-stopper.

The definition of 'quantized' is generally 'using integer arithmetic only'.  The point is often to ensure portability to integer-arithmetic-only hardware, and to provide a starting point for such ports.  So there is a real motivation for aiming for 100% integer-only arithmetic, regardless of performance considerations.

On the other hand, it is desirable to have quantized op implementations to fill functional gaps even if they are not right away 100% integer-only.  That unblocks other aspects of quantization work.  Otherwise, it's not always easy to revert to float for one op whose quantized coverage is not ready.

That is to say that this PR is fine as-is, with:
  - Either a TODO about this current float usage, noting that it would be desirable to convert to 100% integer-only in the future,
  - Or, if you have time, ideally, fix that as part of this PR.

If you're interested in removing the float usage here, please refer to other existing quantized ops in these optimized_ops.h and reference_ops.h files.  See MultiplyByQuantizedMultipler* and RoundingDivideByPOT. This shouldn't be hard and would make this PR even better ;-)